### DISCUSSION TITLE
homebrew-cask-drivers API

### DISCUSSION THREAD

**Discussion by ahousseini** [Homebrew/discussions#825] [Casks]
Hi

I couldn't find any information about this in the API [documentation](https://formulae.brew.sh/docs/api/). Are the casks in [homebrew-cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers) not available through API?

**Comment by SMillerDev** [comment]
They are not. Just core, linuxbrew-core and the main casks repo.
